### ISSUE TITLE
Vulnerability in 1.26.1

### ISSUE THREAD

**Issue by 5049504F** [Microsoft/vscode#56876]
VSCode Version: 1.26.1
OS Version: Windows
I've found a vulnerability in version 1.26.1 which allows attackers to compromise Windows systems where vscode is running.

Could a maintainer reply this issue with an email so that I can give further information ? 
@egamma I´ve seen that you were the responsible for the last vulnerability reported in this project, just give me a heads up if you are still active in this project.

**Comment by egamma** [comment]
@5049504F yes, I´m still active, please provide me with further information at: egamma at microsoft dot com

**Comment by 5049504F** [comment]
Hi @egamma , mail has been sent ! Just let me know if you got it !

**Comment by egamma** [comment]
@5049504F I´m sorry, but I´ve checked my mail inbox again and couldn´t find a mail related to this issue, when did you sent the mail? If my spam filter blocked it, can you zip up the contents and send it to me again. Sorry for the hassle.

**Comment by 5049504F** [comment]
@egamma the mail was sent yesterday at 8.40 pm from a @protonmail account with the subject "Vulnerability in VS Code". I will sent you the email again without file attached.

**Comment by egamma** [comment]
@5049504F pls let me know when you have done so I´ve checked again and I haven´t received a mail from this account yet.
